vpr/src/base/stats.cpp

```diff
@@ -364,9 +366,15 @@ void print_wirelen_prob_dist() {
                 VTR_LOG("Realloc'ing to increase 2-pin wirelen prob distribution array.\n");
                 incr = index - prob_dist_size + 2;
                 prob_dist_size += incr;
+
+                float temp[prob_dist_size];
```

Comment:
Even better change: we could just make this a std::vector and use vector operations.